### PR TITLE
Updated docs with HEALTHCHECK wait strategy

### PR DESCRIPTION
**Pull request by kfrajtak**
## What does this PR do?

Adds a section to docs.

### CODE REVIEW

File: docs/api/wait_strategies.md [hunk 1]
```diff
@@ -10,3 +10,22 @@ _ = Wait.ForUnixContainer()
   .UntilOperationIsSucceeded(() => true, 1)
   .AddCustomWaitStrategy(new MyCustomWaitStrategy());
 ```
+
+## Wait for healthy container
```
Comment (HofmeisterAn) [inline docs/api/wait_strategies.md:14]: Due to nested code block I cannot add a suggestion (at least I do not know how it works 😅). WDYT about something like:

````
## Wait until the container is healthy

If the Docker image supports Dockers's [HEALTHCHECK](docker-docs-healthcheck) feature, like the following configuration:

```Dockerfile
FROM alpine
HEALTHCHECK --interval=1s CMD test -e /healthcheck
```

You can leverage the container's health status as your wait strategy to report readiness of your application or service:

```csharp
_ = new TestcontainersBuilder<TestcontainersContainer>()
  .WithWaitStrategy(Wait.ForUnixContainer().UntilContainerIsHealthy())
  .Build();
```

[docker-docs-healthcheck]: https://docs.docker.com/engine/reference/builder/#healthcheck
````

File: docs/api/wait_strategies.md [hunk 2]
```diff
@@ -10,3 +10,22 @@ _ = Wait.ForUnixContainer()
...
+```
+
+You can use the `UntilContainerIsHealthy` to check that the container is fully ready.
+
+```csharp
+_ = new TestcontainersBuilder<TestcontainersContainer>()
+  .WithImage(imageName)
+  .WithWaitStrategy(Wait.ForUnixContainer().UntilContainerIsHealthy())
+  .Build();
+```
```
Comment (HofmeisterAn) [inline docs/api/wait_strategies.md]:
~~~suggestion

## Wait until the container is healthy

If the Docker image supports Dockers's [HEALTHCHECK](docker-docs-healthcheck) feature, like the following configuration:

```Dockerfile
FROM alpine
HEALTHCHECK --interval=1s CMD test -e /healthcheck
```

You can leverage the container's health status as your wait strategy to report readiness of your application or service:

```csharp
_ = new TestcontainersBuilder<TestcontainersContainer>()
  .WithWaitStrategy(Wait.ForUnixContainer().UntilContainerIsHealthy())
  .Build();
```

[docker-docs-healthcheck]: https://docs.docker.com/engine/reference/builder/#healthcheck

~~~

**Review by HofmeisterAn — APPROVED** [review]
Thanks for keeping the docs up to date.

**Comment by kfrajtak** [comment]
thanks @HofmeisterAn, when will be a NuGet package released?

**Comment by HofmeisterAn** [comment]
No ETA yet. I would like to include some more improvements first. Especially, a new version of Docker.DotNet.

**Pull request merged**